doc/sphinx/reactors.rst

```diff
@@ -195,3 +181,230 @@ temperature:
+
+Plug-Flow Reactors are often used to simulate ignition delay times, emission formation, and catalytic processes.
+
+The governing equations of Plug-Flow Reactors are [KCG2003]_:
+
+- Mass conservation:
+
+ .. math:: \frac{d(\rho u A)}{dz} =  P' \sum_k \dot{s}_k W_k
+
+ where `u` is the axial velocity in (m/s) and `P'` is the chemically active channel perimeter in (m) (chemically active perimeter per unit length).
+
+- Continuity equation of species `k`:
+
+ .. math:: \rho u \frac{d Y_k}{dz} + Y_k P' \sum_k \dot{s}_k W_k =
+      \dot{\omega}_k W_k + P' \dot{s}_k W_k
+
+- Energy conservation:
+
+ .. math:: \rho u A c_p \frac{d T}{d z} =
+     - A \sum_k h_k \dot{\omega}_k W_k
+     - P' \sum_k h_k \dot{s}_k W_k
+     + U P (T_w - T)
+
+ where `U` is the heat transfer coefficient in (W/m/K), `P` is the perimeter of the duct in (m), and `T_w` is the wall temperature in (K). Kinetic and potential energies are neglected.
+
+- Momentum conservation in the axial direction:
+
+ .. math:: \rho u A \frac{d u}{d z} + u P' \sum_k \dot{s}_k W_k =
+     - \frac{d (p A)}{dz} - \tau_w P
+
+ where `\tau_w` is the wall friction coefficient (which might be computed from Reynolds number based correlations).
+
+Even though this problem extends geometrically in one direction, it can be modeled via zero-dimensional reactors: Due to the neglecting of diffusion, downstream parts of the reactor have no influence on upstream parts. Therefore, PFRs can be modeled by marching from the beginning to the end of the reactor.
+
+Cantera does not (yet) provide dedicated class to solve the PFR equations (The ``FlowReactor`` class is currently under development). However, there are two ways to simulate a PFR with the reactor elements previously presented. Both rely on the assumption that pressure is approximately constant throughout the Plug-Flow Reactor and that there is no friction. The momentum conservation equation is thus neglected.
+
+
+PFR Modeling as a Series of CSTRs
+*********************************
+
+The Plug-Flow Reactor is spatially discretized into a large number of axially distributed volumes. These volumes are modeled to be steady-state CSTRs.
+
+In cantera, it is sufficient to consider a single reactor and march it forward in time, because there is no information traveling upstream. The mass flow rate `\dot{m}` through the PFR enters the reactor from an upstream reservoir. For the first reactor, the reservoir conditions are the inflow boundary conditions of the PFR. By performing a time integration as described in `Continuously Stirred Tank Reactor`_ until the state of the reactor is converged, the steady-state CSTR solution is computed. The state of the CSTR is the inlet boundary condition for the next CSTR downstream.
+
+An example for this procedure can be found in `pfr.py <cython/examples/reactors_pfr.html>`_ and `surf_pfr.py <cython/examples/reactors_surf_pfr.html>`_.
```

Comment:
I think it should to be explained that the only reason to use this approach as opposed to the Lagrangian one is if you need to include surface species, because the system of equations ends up being a DAE system instead of an ODE system that the normal ReactorNet model works just fine with.